index.html

```diff
@@ -18,16 +21,178 @@
             </div>
         </nav>
     </header>
-    <!-- END NAVBAR SECTION-->
+    <!-- End NAVBAR SECTION-->
 
-    <!-- START HEADLINE SECTION-->
+    <!-- Start HEADLINE SECTION-->
     <section class="headline">
         <div class="content">
             <h1 class="author">Thiri Htet Htet Aung</h1>
             <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
             <button type="button" class="collab-btn">Start collaboration</button>
         </div>
     </section>
-    <!-- END HEADLINE SECTION -->
+    <!-- End HEADLINE SECTION -->
+
+    <!-- Start WORK SECTION -->
+    <section class="works">
+        <!-- start title -->
+        <div class="title">
+            <h1>Projects</h1>
```

Comment:
In order to have a well ranking of content, it is advisable to have just one `h1` throughout a website which you have done right at the headline section. Every other opening heading should at least be an `h2`. This also makes a website easily navigatable and readable.

I recommend using an html `h2` tag here.